heartbeat/podman-etcd

```diff
@@ -1063,19 +1063,19 @@ reconcile_member_state()
 	fi
 
 	if [ -n "$learner_member_id" ]; then
-		promote_learner_member "$learner_member_id"
-		return $?
+		if ! promote_learner_member "$learner_member_id"; then
+			return $?
```

Comment:
I'm worried that this returns and will never retry. (And that we'll not run into this in testing because our etcds are generally very small for new clusters).